src/VirtualMonoRepo/Component.template.md

```diff
@@ -0,0 +1,15 @@
+# List of components
+
+To enable full offline source-building of the VMR, we have no other choice than to synchronize all the necessary code into the VMR. This also includes any code referenced via git submodules. More details on why and how this is done can be found here:
```

Comment:
I'm not a fan of this content being duplicated here as well as the main readme.  IMO it should be pulled out of the readme.